tests/components/DrugOrderEntry/index.test.jsx

```diff
@@ -188,22 +192,25 @@ describe('behaviour when the length of the unsaved draft orders table is more th
       deleteDraftOrder: jest.fn(),
       deleteAllDraftOrders: jest.fn(),
       drug: "abc-e345-thed-uuid2345",
+      dateFormat: 'DD-MMM-YYYY HH:mm',
     };
     const wrapperOne = shallow(<SearchAndAddOrder {...propsOne} store={store} />);
     const propsTwo = {
       outpatientCareSetting:{
-        uuid:{}
+        uuid: '',
+        display: '',
       },
       inpatientCareSetting: {
-        uuid: {}
+        uuid: '',
+        display: '',
       },
       getPastOrders: jest.fn(),
       draftOrders: [
         {dose: '', doseUnits: ''}
       ],
       setOrderAction: jest.fn(),
       location:{
-        search:()=>{}
+        search: '',
```

Comment:
But why is it changed to a string when `location.search()` is supposed to be a function

Reply:
I believe the `location.search` property is a string as defined [HERE](https://www.w3schools.com/jsref/prop_loc_search.asp).
Also the Proptypes of the component defines it as 
```
location: PropTypes.shape({
    search: PropTypes.string,
  }).isRequired,
```

Reply:
Ok cool.